tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -10,21 +10,22 @@ else
 fi
 
 MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
-DEVICE_MEMORY_LIMIT="25GB"
+DEVICE_MEMORY_LIMIT="15GB"
```

Comment:
Was this change intentional?

Reply:
Yes, based on the idea of using the new Dask JIT spilling w/ 32 GB cards.

I'm not sure though that it should be the default setting..

Reply:
We initially wrote this for a dgx-2 with 16 32B cards. Unsure what this setting is for. That said I think we should focus on stability for the defaults than performance so a lower default looks fine. 

CC: @beckernick  for thoughts.

Reply:
15GB is the recommendation for running SF1K on a DGX-1. I would recommend we keep the existing default, but don't feel strongly.